File: Hospital_Project/Hospital_Project/Models/Patient.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Hospital_Project.Models
{
    public partial class Patient
    {
        public int Id { get; set; }
        public int AppointmentId { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public string PhoneNumber { get; set; }
        public int DepartmentId { get; set; }
        public int DoctorId { get; set; }
        public string PatientStatus { get; set; }
        public string Comments { get; set; }
        public int Refer { get; set; }

        public virtual Department Appointment { get; set; }
        public virtual Department Department { get; set; }
        public virtual Doctor Doctor { get; set; }
        public virtual Doctor ReferNavigation { get; set; }
    }
}

File: Hospital_Project_FE/src/app/Models/patient.ts
import { Doctor } from "./doctor";

export class Patient {
     id:number = 0;
     appointmentId:number=0;
     name:string="" 
     address:string=""
     phoneNumber:string=""
     departmentId:number=0
     department:string=""
     doctorId:number=0;
     doctor:string="" 
     status:any =""
     comments:any
     refer:number=0
     referName:string=""
     userId:any=""
     doctors:Doctor= new Doctor();
}
